wp-content/themes/reconasia/assets/_scss/pages/archive.scss

```diff
@@ -1,6 +1,96 @@
 @use '../abstracts' as *;
 @use '../components/entry-header';
 
+.archive {
+  &__featured {
+    position: relative;
+    margin: 0 calc(var(--container-padding) * -1);
+    padding: 0 var(--container-padding);
+    background: $color-bg-dark-200;
+    @include shadow('lg');
+
+    @include breakpoint('small') {
+      margin: 0 auto;
+      padding: 0;
+    }
+
+    @include breakpoint('medium') {
+      background: transparent;
+      box-shadow: none;
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      top: -20px;
+      right: calc(var(--container-padding) * -1);
+      width: calc(100% - var(--container-padding));
+      height: 50px;
+      background: #fff;
+    }
+
+    &-label {
+      position: relative;
+      z-index: 1;
+      display: flex;
+      align-items: flex-end;
+      width: fit-content;
+      height: rem(30);
+      margin-left: 0;
+      padding: rem(12) rem(32) 0 0;
+      color: $color-white-190;
+      @extend %text-style-label-small;
+      text-transform: uppercase;
+      background-color: $color-bg-dark-200;
+      @include shadow('lg');
+      clip-path: polygon(0 0, 85% 0, 100% 100%, 0% 100%);
+
+      @include breakpoint('small') {
```

Comment:
The clip-path breaks at around ~974px. To resolve this we can adjust the clip-path ever so slightly to account for larger screen sizes: 

clip-path: polygon(0 0, 85% 0, 100% 101%, 0% 101%);